### PR TITLE
Update 02_headers.md

### PR DESCRIPTION
**Pull request by nemophila1998**
Hello!
Thank you very much for your fun work!
Very good lesson for beginners to learn GitHub, so please fix to follow instructions smoothly.

### CODE REVIEW

File: responses/02_headers.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The toolbar isn't available everywhere. When you edit a file, you must type the
       ```
 {% else %}
 1. In this pull request, click the **Files changed** tab.
-1. In the upper right corner of the file view, click the **small pencil** ✏️ or **hamburger ...** icon to edit the file titled `_includes/01-name.md`.
+1. To edit the file titled `_includes/01-name.md`, click the **small pencil** ✏️ or select "Edit file" from the **meatballs ...** menu in the upper right corner of the file view.
```
Comment (brianamarie) [inline responses/02_headers.md:57]:
```suggestion
1. To edit the file titled `_includes/01-name.md`, click the **small pencil** ✏️ or select "Edit file" from the three dotted **...** menu in the upper right corner of the file view.
```

**Review by brianamarie — CHANGES_REQUESTED** [review]
Hello @nemophila1998! Thank you for this improvement.

I've suggested a change from meatball, and also removing hamburger, to something hopefully more universally descriptive. What do you think?

**Comment by nemophila1998** [comment]
Hello.
I think it's good. Thanks.

**Review by brianamarie — APPROVED** [review]
Thank you so much, @nemophila1998!

**Pull request merged**